File: csm/paths.py
import os

def config_dir():
    env = os.environ.get("CLAUDE_CONFIG_DIR")
    if env:
        return env
    return os.path.join(os.path.expanduser("~"), ".claude")

def projects_dir():
    return os.path.join(config_dir(), "projects")

def claude_command():
    return "claude"

def index_db_path():
    return os.path.join(config_dir(), "csm-index.db")

def favorites_path():
    return os.path.join(config_dir(), "csm-fav.json")

def trash_meta_path():
    return os.path.join(config_dir(), "csm-trash.json")

def trash_dir():
    return os.path.join(projects_dir(), "_trash")

File: lib/paths.js
const os = require("os");
const path = require("path");

function configDir() {
  return process.env.CLAUDE_CONFIG_DIR || path.join(os.homedir(), ".claude");
}
function projectsDir() {
  return path.join(configDir(), "projects");
}
function indexCache() {
  return path.join(configDir(), "csm-index.json");
}
function favPath() {
  return path.join(configDir(), "csm-fav.json");
}
function trashMeta() {
  return path.join(configDir(), "csm-trash.json");
}
function trashDir() {
  return path.join(projectsDir(), "_trash");
}
function sessionsDir() {
  return path.join(configDir(), "sessions");
}
function codexIndexCache() {
  return path.join(configDir(), "csm-codex-index.json");
}

module.exports = { configDir, projectsDir, indexCache, favPath, trashMeta, trashDir, sessionsDir, codexIndexCache };
